tutorindigo/templates/indigo/lms/templates/index.html

```diff
@@ -0,0 +1,78 @@
+<%page expression_filter="h"/>
```

Comment:
Which part of this file was modified, compared to the original? We should add a comment to indicate where it was changed.

Reply:
Added